public/app/plugins/panel/alertlist/UnifiedAlertList.tsx

```diff
@@ -159,7 +159,9 @@ export function UnifiedAlertList(props: PanelProps<UnifiedAlertListOptions>) {
     if (!promRulesRequests.loading) {
       fetchPromAndRuler({ dispatch, limitInstances, matcherList, dataSourceName, stateList });
     }
+
     const sub = dashboard?.events.subscribe(TimeRangeUpdatedEvent, () => {
+      console.log('TimeRangeUpdatedEvent');
```

Comment:
```suggestion
```

Reply:
Actually changes in this entire file are to be reverted :)

Reply:
yea, reverted these changes :)